frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.scss

```diff
@@ -4,3 +4,7 @@
   justify-content: space-between;
   align-items: center;
 }
+
+.odc-revisions-link {
+  font-size: small;
+}
```

Comment:
It looks like this is not necessary, it doesn't change the resource link size even if its removed.

Reply:
I checked it again, the font looks larger without the css.
I'll check it once again.

Reply:
It didn't make a difference afterall. I've updated the code.